kong/plugins/apifortress/fortress_serializer.lua

```diff
@@ -61,10 +55,11 @@ function _M.serialize(ngx)
     failed = true
   end
   local uri = ngx.ctx.api.upstream_url
-  if string.ends(uri,"/") and string.starts(ngx.var.request_uri, "/") then
+  if stringy.endswith(uri,"/") and stringy.startswith(ngx.var.request_uri, "/") then
     uri = string.sub(uri,0,string.len(uri)-1)
```

Comment:
0 is not a proper index in Lua. Index starts from 1. Additionally; negative indices count from the end.

So assuming you want to drop the trailing slash you'd write;
```lua
  uri = uri:sub(1, -2)
```